nearcore/src/state_sync.rs

```diff
@@ -194,79 +233,110 @@ async fn state_sync_dump(
                 epoch_id,
                 epoch_height,
                 sync_hash,
-                parts_dumped,
+                parts_dumped: _,
```

Comment:
If `parts_dumped` isn't read, maybe it doesn't need to be stored in `InProgress`?

Reply:
will it cause backward compatibility issues? maybe not, since we will mount new snapshot and start node fresh.